core_backend/app/urgency_detection/models.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+
+from sqlalchemy import JSON, Boolean, DateTime, Integer, String, select
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from sqlalchemy.sql.schema import ForeignKey
+
+from ..models import Base, JSONDict
+from .schemas import UrgencyQuery, UrgencyResponse
+
+
+class UrgencyQueryDB(Base):
+    """
+    Urgency query database model.
+    """
+
+    __tablename__ = "urgency-queries"
```

Comment:
We should standardize the names of our tables. For example we use `content` in singular but `urgency-queries` in plural here.

Reply:
This DB cleanup PR #175 will eventually be merged to main so the table names here should match it. So yeah agreed, singular names.